UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/Helpers/CommonHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Extensions.Logging;
+using Polly;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.Common.Logging;
+
+namespace UKHO.ExternalNotificationService.SubscriptionService.Helpers
+{
+    public static class CommonHelper
+    {
+        public static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(ILogger logger, EventIds eventId, int retryCount, double sleepDuration)
+        {
```

Comment:
No need to pass EventIds as method parameter as its always RetryHttpClientD365CallbackRequest

Reply:
done